torch/_dynamo/eval_frame.py

```diff
@@ -1240,6 +1240,7 @@ def patch():
             opt._cuda_graph_capture_health_check = disable(
                 opt._cuda_graph_capture_health_check
             )
+            # Default zero_grad sets attributes to None, which cannot be optimized
```

Comment:
I think the main reason is that Dynamo spends just way too much time tracing zero_grad, and at the end, there is nothing to optimize.